### ISSUE TITLE
Impossible to run

### ISSUE THREAD

**Issue by nexttoyou** [mkellsy/homebridge-config-ui#1]
Unfortunately, the plugin just does not want to cooperate. Are there any hidden prerequisites or whatsoever perhaps forgotten in the documentation? Until now I never had such troubles to run a homebridge module but I cannot exclude any error on my side of course (it’s quite hot here …). Any help very appreciated. Here’s the error log after the successful login:

```
Error: Not Found
    at /usr/local/lib/node_modules/homebridge-config-ui/app.js:76:15
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:317:13)
    at /usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:335:12)
    at next (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:275:10)
    at /usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:635:15
    at next (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:260:14)
    at Function.handle (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:174:3)
    at router (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:317:13)
    at /usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:335:12)
    at next (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/express/lib/router/index.js:275:10)
    at SessionStrategy.strategy.pass (/usr/local/lib/node_modules/homebridge-config-ui/node_modules/passport/lib/middleware/authenticate.js:325:9)
```

What exactly is not found and why?

Many thanks!

**Comment by mkellsy** [comment]
This looks like it is failing on the express request handler.

Did you get any errors when installing the plugin?
Also what version of node are you running?

**Comment by mkellsy** [comment]
I dug into this and I found what's happening. I had a typo in the login fail redirect. This is because there is no auth.json file in your homebridge user files.

To address this mistake I added a check for this file. If the file doesn't exists, I create it with a default username and password.

The current workaround is. add the auth.json to your homebridge user files. This is the same place as your config.json file.

Enter at least 1 user to this file;
`[{
    "id": 1,
    "username": "admin",
    "password": "admin",
    "name": "Administrator"
}]`

This more streamlined in version 1.1 (releasing soon)
